pkg/controller/test_helper.go

```diff
@@ -40,13 +42,26 @@ func CreateLoadtest(clientSet clientSetV.Clientset, pods int32, name, testFile,
 		}
 	}
 
+	loadtestSpec, err := backends.BuildLoadTestSpecByBackend(
+		loadTestType,
+		false,
```

Comment:
This very hard to read, not sure maybe leave the old version with `ltObj.Spec.` ?

Reply:
We cannot define `ltObj.Spec.*` explicitly here; we need to call `backends.BuildLoadTestSpecByBackend()` to obtain the default master/worker config from it.

Reply:
The test was previously working because `SetDefaults()` was updating the Spec with it.  
Now that we remove `SetDefaults()`, the spec needs to have MasterConfig and WorkerConfig defined somewhere; hence the use of `backends.BuildLoadTestSpecByBackend()` here.